scripts/validate_unwanted_patterns.py

```diff
@@ -337,17 +336,23 @@ def main(
                     )
                 )
 
+    FILE_EXTENSIONS_TO_CHECK: FrozenSet[str] = frozenset(
```

Comment:
```suggestion
    FILE_EXTENSIONS_TO_CHECK = frozenset(
```

Mypy will infer this from the right hand side